tests/local.rs

```diff
@@ -332,162 +381,164 @@ async fn resolve_execute_async_value(mut c: Client, port: u16) -> Result<(), err
 
 mod firefox {
     use super::*;
+    use common::*;
     #[test]
     #[serial]
     fn navigate_to_other_page() {
-        local_tester!(goto, "firefox")
+        tester_local!(goto, "firefox");
```

Comment:
These diffs seem unnecessary. Can we just have the macro be called `tester_local` the way it was?